src/lib/bootloader/systeminfo.rb

```diff
@@ -108,6 +142,30 @@ def efi_mandatory?
         Yast::Arch.aarch64 || Yast::Arch.arm || Yast::Arch.riscv64
       end
 
+      # Check if console settings are supported
+      #
+      # param bootloader_name [String] bootloader name
+      # @return [Boolean] true if supported
+      def console_supported?(bootloader_name)
+        !Yast::Arch.s390 && bootloader_name != "grub2-bls"
```

Comment:
@aplanas Just to be sure grub2-bls does not support neither graphical console neither serial console? Especially later one can be a bit problematic on servers. Or it works in different way and is just not configurable and detect serial console automatic?

Reply:
It is supporting both